### ISSUE TITLE
Appbar "blocks" search reccomendations in 2019 (change title please)

### ISSUE THREAD

**Issue by HAYDENCRAFT2008** [lightbeam24/CustomTube#180]
![image](https://github.com/lightbeam24/CustomTube/assets/141458603/276c31a3-80c4-4b67-95d3-c63b4e7a624f)

(The Title is terrible I know)

but basically, only on the kevlar layout 2019, the appbar get in the way of some of the search reccomendations when on the search (does not happen on 2020 Kevlar nor 2021) This is one of the only reasons why I don't use 2019.

**Comment by HAYDENCRAFT2008** [comment]
![image](https://github.com/lightbeam24/CustomTube/assets/141458603/715f3321-daac-4dc5-b49d-bfd67f3b08af)
2020 for comparison

**Comment by lightbeam24** [comment]
Yeah after using the 2019 layout with the appbar more, this is a pretty big issue that I should definitely fix for the next update.
